cl/assets/react/tag-page.css

```diff
@@ -0,0 +1,85 @@
+
+.editor-toolbar {
```

Comment:
Can you skim this, Bill, and see if anything looks obsolete to you?

Reply:
yes. good eye

Reply:
√